balboa-http/src/main/scala/com/socrata/balboa/server/MetricsServlet.scala

```diff
@@ -69,15 +61,12 @@ class MetricsServlet extends JacksonJsonServlet
       return unacceptable.result
     }
 
-    val begin = System.currentTimeMillis()
-
     val range = DateRange.create(period, ServiceUtils.parseDate(date).getOrElse {
       contentType = json
       return malformedDate(date).result
     })
 
-    try
-    {
+    timer("period queries") {
```

Comment:
Why are you starting this here? It looks like previously the beginning time was above the range block?